app/commands/v2/publish.rb

```diff
@@ -6,8 +6,11 @@ def call
 
         publish_edition
 
-        after_transaction_commit do
-          send_downstream(document.content_id, document.locale, update_type)
+        if downstream
+          after_transaction_commit do
+            send_downstream_live
+            send_downstream_draft
```

Comment:
we could just do this only if access limits exist. Code might be a little ugly to do it as we'd have to check for existence pre publish_edition, but would save an unnecessary represent downstream call